### PR TITLE
Feature rebuild user interface

### PR DESCRIPTION
**Pull request by Szymiks**
Resolves: https://github.com/Code-Poets/sheetstorm/issues/103

### CODE REVIEW

File: users/static/users/center_style.css [hunk 1]
```diff
@@ -1,13 +1,101 @@
-table {
-  margin-left:auto;
-  margin-right:auto;
+body{
```
Comment (kbeker) [inline users/static/users/center_style.css]: Please add space between name and style declaration

File: users/static/users/center_style.css [hunk 2]
```diff
@@ -1,13 +1,101 @@
...
+    background-color: #f2f2f2;
+    width:100%;
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
 }
 
-th, td.padding {
-  padding: 15px;
+#menu{
```
Comment (kbeker) [inline users/static/users/center_style.css]: Same here

File: users/static/users/center_style.css [hunk 3]
```diff
@@ -1,13 +1,101 @@
...
+  position: fixed;
+  top:0;
+  color: white;
+  transition: 0.5s;
+
+  -webkit-box-shadow: 2px 0px 5px 0px rgba(0,0,0,0.77);
+  -moz-box-shadow: 2px 0px 5px 0px rgba(0,0,0,0.77);
+   box-shadow: 2px 0px 5px 0px rgba(0,0,0,0.77);
+ }
+#menu a{
```
Comment (kbeker) [inline users/static/users/center_style.css]: And here

File: users/static/users/center_style.css [hunk 4]
```diff
@@ -1,13 +1,101 @@
...
+  #menu {
+    margin-left: 0;
+  }
+
+
+  #wrapper.toggled #menu {
+    margin-left: -35rem;
+  }
+}
+
```
Comment (kbeker) [inline users/static/users/center_style.css]: Please add spaces between names and style declarations

File: users/static/users/center_style.css [hunk 5]
```diff
@@ -1,13 +1,101 @@
-table {
-  margin-left:auto;
-  margin-right:auto;
+body{
+    background-color: #f2f2f2;
+    width:100%;
```
Comment (kbeker) [inline users/static/users/center_style.css]: MIssing space between value and `:`

File: users/static/users/center_style.css [hunk 6]
```diff
@@ -1,13 +1,101 @@
...
+#menu{
+  margin-left: -15rem;
+  width: 15%;
+  height: 100%;
+  -webkit-transition: margin .25s ease-out;
+  -moz-transition: margin .25s ease-out;
+  -o-transition: margin .25s ease-out;
+  background-color: #2c77bc;
+  position: fixed;
+  top:0;
```
Comment (kbeker) [inline users/static/users/center_style.css]: MIssing space between value and `:`

File: users/static/users/center_style.css (hunk 4 above)
Comment (kbeker) [reply users/static/users/center_style.css]: Please also add everywhere where it is needed space between argument and `:`

File: users/templates/base.html [hunk 7]
```diff
@@ -6,56 +6,154 @@
...
+    avatar_div_styles.marginLeft = "0";
+    for (i = 0; i < text_div_styles.length; i++) {
+        text_div_styles[i].style.display = "block";
+    }
+  }
+}
+
+
+
+</script>
```
Comment (kbeker) [inline users/templates/base.html]: Please move this script to `.js` file. HTML files should be have only html "code" :D

File: users/static/users/center_style.css (hunk 1 above)
Comment (Szymiks) [reply users/static/users/center_style.css]: Done

File: users/static/users/center_style.css (hunk 2 above)
Comment (Szymiks) [reply users/static/users/center_style.css]: Done

File: users/static/users/center_style.css (hunk 3 above)
Comment (Szymiks) [reply users/static/users/center_style.css]: Done

File: users/static/users/center_style.css (hunk 4 above)
Comment (Szymiks) [reply users/static/users/center_style.css]: Done

File: users/static/users/center_style.css (hunk 5 above)
Comment (Szymiks) [reply users/static/users/center_style.css]: Done

File: users/static/users/center_style.css (hunk 6 above)
Comment (Szymiks) [reply users/static/users/center_style.css]: Done

File: users/templates/base.html (hunk 7 above)
Comment (Szymiks) [reply users/templates/base.html]: Done

**Comment by Szymiks** [comment]
Close this PR because after rebase I have got problems. I created it again, it should work better than before.

**Pull request closed without merging**